steps/bitbucket-server-post-build-status/0.0.3/step.yml

```diff
@@ -0,0 +1,73 @@
+title: Bitbucket server post build status
+summary: |
+  Post build status to bitbucket server
+description: |
+  Post build status to bitbucket server
+website: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status
+source_code_url: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status
+support_url: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status/issues
+published_at: 2019-12-18T14:50:01.626161+01:00
+source:
+  git: https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status.git
+  commit: 333cb502868ec24abb363fb733434ded50e85744
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: false
+inputs:
+- bitbucket_server_domain: null
+  opts:
+    is_required: true
+    summary: Full domain name without protocol eg 'my-domain.com'
+    title: Bitbucket Server domain name
+- bitbucket_server_username: null
+  opts:
+    is_required: true
+    summary: Username used to make REST calls
+    title: Bitbucket Server username
+- bitbucket_server_password: null
+  opts:
+    is_required: true
+    is_sensitive: true
+    summary: Password for username
+    title: Bitbucket Server password
+- build_is_in_progress: false
```

Comment:
In case a fixed list of inputs are accepted, use `value_options` to list them (e.g. https://github.com/bitrise-steplib/steps-gitlab-status/blob/94d495726810cad4963dde32267af6cead580f02/step.yml#L78); Use "yes" and "no" instead of true/false.

Reply:
Fixed